From 99204cc1a173c4dc4b0df40aeb00dab00c9ed1f0 Mon Sep 17 00:00:00 2001
From: Jonathan Lorimer <jonathan_lorimer@mac.com>
Date: Mon, 25 Oct 2021 00:17:57 -0400
Subject: [PATCH] make types bold

---
 lua/nordbones/init.lua | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lua/nordbones/init.lua b/lua/nordbones/init.lua
index 1d4899d..803e20b 100644
--- a/lua/nordbones/init.lua
+++ b/lua/nordbones/init.lua
@@ -10,6 +10,6 @@ return lush.extends({ specs }).with(function()
 		Identifer { fg = p.derived.fg },
 		Function { fg = p.base.nord8 },
 		Statement { fg = p.base.nord9 },
-		Type { fg = p.base.nord10 },
+		Type { fg = p.base.nord10, gui = "Bold" },
 	}
 end)
--
2.33.0

